yabt/buildcontext.py

```diff
@@ -464,11 +464,28 @@ def build(target: Target):
                 if run_tests and 'testable' in target.tags:
                     if self.conf.no_test_cache or not test_cached:
                         logger.info('Testing target {}', target.name)
-                        test_start = time()
-                        self.test_target(target)
+                        retries = target.props.retries or self.conf.test_retries
+                        if retries < 0:
+                            raise ValueError(
+                                'Retries value must be positive: got {}'
+                                .format(retries))
+                        test_start = 0
+                        fails = 0
+                        while retries - fails >= 0:
```

Comment:
@itamar-resonai Bah... didn't turn out much better...
I need to:
a. increment fails
b. catch exception, 
  b.1 continue if still more retries available
  b.2 raise if no more retries
c. break on success